Mathlib/Algebra/Order/Group/Cone.lean

```diff
@@ -58,6 +58,9 @@ instance GroupCone.instGroupConeClass (G : Type*) [CommGroup G] :
   one_mem {C} := C.one_mem'
   eq_one_of_mem_of_inv_mem {C} := C.eq_one_of_mem_of_inv_mem'
 
+initialize_simps_projections GroupCone (carrier → coe, as_prefix coe)
+initialize_simps_projections AddGroupCone (carrier → coe, as_prefix coe)
```

Comment:
Could you please make the `initialize_simps_projections` + `to_additive` changes a separate PR?